### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by rnshah9**

### CODE REVIEW

File: mayhem/FuzzedDataProvider.h [hunk 1]
```diff
@@ -0,0 +1,397 @@
+//===- FuzzedDataProvider.h - Utility header for fuzz targets ---*- C++ -* ===//
```
Comment (xansec) [inline mayhem/FuzzedDataProvider.h:1]: Please either install latest LLVM directly or build from source.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: mayhem/FuzzedDataProvider.h (hunk 1 above)
Comment (rnshah9) [reply mayhem/FuzzedDataProvider.h]: I just removed usage of FDP entirely.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**